src/index.html

```diff
@@ -7,6 +7,374 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+          class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="catalog">
+        <div class="catalog__cards">
+          <!-- Card #1 -->
+          <article class="catalog__card product-card" data-qa="card">
+            <img
+              src="images/imac.jpeg"
+              alt="iMac"
+              class="product-card__img"
+            >
+
+            <div class="product-card__data">
+              <h2 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+              <p class="product-card__product-number">Код товара: 195434</p>
+            </div>
+
+            <div class="product-card__ratings">
+              <div class="stars stars--4">
```

Comment:
should stars block be as an element of product-card block?

Reply:
Again that's a block on its own) I thinks it's fine, It's not that I do some external geometry, in which case I would have to make it a child